common-rendering/src/editorialPalette.ts

```diff
@@ -18,8 +18,17 @@ type Colour = string;
 
 interface Palette {
 	text: {
+		articleLink: Colour;
+		bylineLightGrayBackground: Colour;
+		bylineDarkGrayBackground: Colour;
+		follow: Colour;
 		headline: Colour;
 		headlineDark: Colour;
+		invertedByline: Colour;
+		invertedFollow: Colour;
+		invertedLink: Colour;
```

Comment:
Does `inverted` mean "dark mode"? Can we use the convention of putting "dark" in the suffix as established elsewhere in this file? E.g. `text.byline` and `text.bylineDark`.